src/ecmascript_simd.js

```diff
@@ -120,6 +120,12 @@ function simdCreate(type) {
   return type.fn.apply(type.fn, lanes);
 }
 
+function simdCreateFromBuffer(type, buffer) {
+  var newValue = type.fn.apply(type.fn, lanes);
+  newValue.s_ = new type.view(buffer);
```

Comment:
If someone does this:
```
  var i4  = SIMD.Int32x4(0x7fc00000,0x7fe00000,0x7ff00000,0x7ff80000);
  var f4  = SIMD.Float32x4.fromInt32x4Bits(i4);
  f4.s_[0] = 1.0;
```
`i4` will change as well.  But that's not a 'legal' use of the polyfill.  If people start messing with the internals of the polyfill they shouldn't expect things to work.  Do you agree?